Assets/MRTK/Providers/XRSDK/XRSDKDeviceManager.cs

```diff
@@ -59,7 +59,8 @@ public override void Update()
                     return;
                 }
 
-                InputDevices.GetDevicesWithCharacteristics(InputDeviceCharacteristics.Controller | InputDeviceCharacteristics.HandTracking, inputDevices);
+                // Need to make it so we get devices that are either controllers or handtracking, the old commented code only got devices that met both requirements
+                InputDevices.GetDevicesWithCharacteristics(InputDeviceCharacteristics.Controller, inputDevices);
```

Comment:
I wonder if we want a second pass through to get `HandTracking` devices explicitly. Out of scope for this PR, but maybe something for future investigation. I believe HL2 currently marks hands with both characteristics, so we're fine there.

Reply:
Because GetDevicesWithCharacteristics fetches devices that exactly match the bitmask passed through, I altered the code so that we get devices that are either controllers, handtracking, or both